db/migrations/202407151352_autoincrement.go

```diff
@@ -0,0 +1,80 @@
+package migrations
+
+import (
+	_ "embed"
+
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+// This migration (inside a DB transaction),
+// - Adds AUTOINCREMENT to the primary key of:
+// - apps, app_permissions, request_events, response_events, user_configs
+//
+// request_events and response_events are not critical (and also payments are dropped in the same release)
+// so we just drop those tables and re-create them.
+var _202407151352_autoincrement = &gormigrate.Migration{
+	ID: "202407151352_autoincrement",
+	Migrate: func(db *gorm.DB) error {
+
+		if err := db.Transaction(func(tx *gorm.DB) error {
```

Comment:
gorm migrations are not by default in a transaction?

or should we split up this transaction into several ones for each table? seems a bit a lot for the one transaction

Reply:
@bumi the default is off - we can turn it on in the gorm settings

If we split this into multiple transactions, then if any of them fail we will have trouble re-running the migration (next time the migration will fail because the DB state is not expected).

I would instead split this into multiple migrations. What do you think?

Reply:
I actually don't think there is much data though - It's only the user configs and apps/app permissions. Everything else is cleared